api/app/db/crud/weather_models.py

```diff
@@ -369,8 +392,7 @@ def get_latest_station_prediction_per_day(session: Session,
         .join(PredictionModel, PredictionModelRunTimestamp.prediction_model_id == PredictionModel.id)\
         .join(subquery, and_(
             WeatherStationModelPrediction.prediction_timestamp == subquery.c.latest_prediction,
-            WeatherStationModelPrediction.station_code == subquery.c.station_code))\
-        .order_by(WeatherStationModelPrediction.update_date.desc())
```

Comment:
I think we can undo this change since it's not the original table we're concerned about here right?

Reply:
Yeah, i was experimenting with performance to see if we could each out a few more milliseconds by not ordering the results. It didn't buy us more than a couple milliseconds so we can definitely revert.